distribution/docker/build.gradle

```diff
@@ -16,20 +15,39 @@ apply plugin: 'elasticsearch.test.fixtures'
 apply plugin: 'elasticsearch.internal-distribution-download'
 apply plugin: 'elasticsearch.rest-resources'
 
-// Define a repository that allows Gradle to fetch a resource from GitHub. This
-// is only used to fetch the `tini` binary, when building the Iron Bank docker image
-// for testing purposes.
+String buildId = System.getProperty('build.id')
```

Comment:
We should use `providers.systemProperty()` here. Do a search to see examples of other usages in our build.

Reply:
Changed to use a provider.